test/actions/profile.spec.js

```diff
@@ -10,7 +10,7 @@ describe('getComments', () => {
     expect(typeof getComments).toBe('function');
   });
 
-  it('should create an appropiate action', () => {
+  it('should create an appropriate action', () => {
```

Comment:
lets get simple in our wording 

```suggestion
  it('should create an action', () => {
```